### DISCUSSION TITLE
WSA Preview Program

### DISCUSSION THREAD

**Discussion by sn-o-w** [microsoft/WSA#200] [General]
I submitted the form for joining WSA Preview Program about two weeks ago and have yet to receive any confirmation email.

Should I have to wait longer? 🤔

**Comment by GuyInCanada** [comment]
you won't get a confirmation email, the update will just show up in Microsoft Store

**Reply by sn-o-w** [reply]
How I’m supposed to know if I’m in WSA Preview Program?

Didn’t see any update in Microsoft Store or something like that. 🤨

**Reply by Darkhost1999** [reply]
> How I’m supposed to know if I’m in WSA Preview Program?

In the end it'll be very difficult to miss.

![amazon-appstore-windows-11-store_hero.jpg](https://github.com/microsoft/WSA/assets/60384196/ef633c22-2564-4d3f-8de3-663a4298aac3)

And I'm having the same issue. Can't join. No email, no update. Nothing.
